inputs/chemistry/athinput.chem_H2

```diff
@@ -0,0 +1,62 @@
+<comment>
+problem   = H-H2 chemistry with analytic solution
+reference = Section 4.1 in Gong et al. 2023, arXiv:2305.04965
+configure = --prob=chem_uniform --chemistry=H2 --ode_solver=forward_euler --eos=isothermal
+
+<job>
+problem_id = chem_H2   # problem ID: basename of output filenames
+
+<output1>
+file_type  = hst       # vtk data dump
+dt         = 0.05      # time increment between outputs
+
+<time>
+cfl_number = 0.3       # The Courant, Friedrichs, & Lewy (CFL) Number
+nlim       = -1       # cycle limit
+tlim       = 50        # time limit, 50 Myr
+
+<mesh>
+nx1        = 4        # Number of zones in X1-direction
+x1min      = -0.5      # minimum value of X1
+x1max      = 0.5       # maximum value of X1
+ix1_bc     = periodic  # ner-X1 boundary flag
+ox1_bc     = periodic  # outer-X1 boundary flag
+
+nx2        = 4        # Number of zones in X2-direction
+x2min      = -0.5      # minimum value of X2
+x2max      = 0.5       # maximum value of X2
+ix2_bc     = periodic  # inner-X2 boundary flag
+ox2_bc     = periodic  # outer-X2 boundary flag
+
+nx3        = 4         # Number of zones in X3-direction
+x3min      = -0.5      # minimum value of X3
+x3max      = 0.5       # maximum value of X3
+ix3_bc     = periodic  # inner-X3 boundary flag
+ox3_bc     = periodic  # outer-X3 boundary flag
+
+<meshblock>
+nx1        =    4
+nx2        =    4
+nx3        =    4
+
+#hydro block is mendatory
+<hydro>
+gamma = 1.666666666666667 # gamma = C_p/C_v
+iso_sound_speed = 1.0     # isothermal sound speed
+sfloor   =   0            # passive scalar floor
+#active   =   background   # "fixed"/"background" for post-processing with/without scalar advection
+
+<problem>
+nH         = 100 #initial density, in units of number density of hydrogen atoms
```

Comment:
add space between `#` and next character

Reply:
space added